src/mono/netcore/sample/.gitignore

```diff
@@ -0,0 +1,4 @@
+wasm-browser/StaticServer/wwwroot/managed
```

Comment:
I guess this should to go `/artifacts` somewhere instead

Reply:
Do you mean serve the static files from there?  We can do that instead I presume instead of using local in tree files.